### ISSUE TITLE
Training Setting

### ISSUE THREAD

**Issue by Nonbiuld** [nickyisadog/latent-diffusion-inpainting#4]
How many epochs did you train? What was the loss value when you stopped training?

**Comment by bowen-upenn** [comment]
Same question here.

**Comment by hnsyq69** [comment]
May I ask if you have solved it? I'd also like to know how many epochs of training to complete. Thank you for your answer.

**Comment by ultiwinter** [comment]
hello, any luck guys?
